topology/probes/netlink/netlink.go

```diff
@@ -92,6 +92,17 @@ type Probe struct {
 	wg       sync.WaitGroup
 }
 
+// VFS describes a vfs fields
```

Comment:
need to add the easyjson tag and to write the common.Getter interface to be sure that we can retrieve and filter field of this structure with gremlin